packages/react-native-bpk-component-calendar/src/common-types.js

```diff
@@ -34,13 +35,26 @@ export const SELECTION_TYPES = {
 };
 
 export type SelectionType = $Keys<typeof SELECTION_TYPES>;
+export type SelectedDatesChanged = ?(Date[]) => mixed;
+export type NativeEvent = SyntheticEvent<$ReadOnly<{| selectedDates: Date[] |}>,>; // eslint-disable-line prettier/prettier
```

Comment:
aren't native dates a number? (timestamp)

Reply:
Oh yeah you're right

Reply:
Also just noticed there is an extra comma at the end.
And you could replace $ReadOnly with `+{| selectedDates: Date[] |}` :)

Reply:
The extra comma gets added by prettier

Reply:
I tried replacing it with `+{| selectedDates: number[] |}`, but that causes a syntax error.